test/hotspot/jtreg/TEST.ROOT

```diff
@@ -48,6 +48,8 @@ requires.extraPropDefns.libs = \
     ../../lib/jdk/test/lib/Container.java
 requires.extraPropDefns.javacOpts = --add-exports java.base/jdk.internal.foreign=ALL-UNNAMED
 requires.extraPropDefns.vmOpts = \
+    -XX:+UnlockDiagnosticVMOptions -XX:+LogVMOutput -XX:-DisplayVMOutput -XX:LogFile=vmprops.flags.final.vm.log \
```

Comment:
The -XX:+UnlockDiagnosticVMOptions is repeated, that shouldn't be necessary. (in either TEST.ROOT)